mmcv/ops/merge_cells.py

```diff
@@ -95,8 +95,18 @@ def _resize(self, x, size):
         elif x.shape[-2:] < size:
             return F.interpolate(x, size=size, mode=self.upsample_mode)
         else:
-            assert x.shape[-2] % size[-2] == 0 and x.shape[-1] % size[-1] == 0
-            kernel_size = x.shape[-1] // size[-1]
+            if x.shape[-2] % size[-2] != 0 or x.shape[-1] % size[-1] != 0:
+                h, w = x.shape[-2:]
+                target_h, target_w = size
+                pad_h = (h // target_h + 1) * target_h - h
+                pad_w = (w // target_w + 1) * target_w - w
```

Comment:
Suppose x.shape[-1] can be divided by size[-1] and is it necessary to still pad the x?